pkg/compose/create.go

```diff
@@ -762,10 +762,6 @@ func buildContainerMountOptions(p types.Project, s types.ServiceConfig, img moby
 						Target:   m.Destination,
 						ReadOnly: !m.RW,
 					}
-					// Avoid mount to be later re-defined
-					l := len(s.Volumes) - 1
-					s.Volumes[i] = s.Volumes[l]
-					s.Volumes = s.Volumes[:l]
```

Comment:
This is an attempt at deleting the i element, but clearly doing this while iterating is a recipe for disaster…

However there was a definite intent there, as the comment seems to imply.
Do we really not know what it’s supposed to do?

Do we have unit tests? 😁